Example/Core/Tests/FIRAnalyticsConfigurationTest.m

```diff
@@ -44,49 +50,69 @@ - (void)testSharedInstance {
 
 /// Test that setting the minimum session interval on the singleton fires a notification.
 - (void)testMinimumSessionIntervalNotification {
+  // Set up the expectation for the notification.
   FIRAnalyticsConfiguration *config = [FIRAnalyticsConfiguration sharedInstance];
-  [config setMinimumSessionInterval:2601];
   NSString *notificationName = kFIRAnalyticsConfigurationSetMinimumSessionIntervalNotification;
-  OCMVerify([self.notificationCenterMock postNotificationName:notificationName
-                                                       object:config
-                                                     userInfo:@{
-                                                       notificationName : @2601
-                                                     }]);
+  [self expectNotificationForObserver:self.observerMock
+                     notificationName:notificationName
+                               object:config
+                             userInfo:@{
+                               notificationName : @2601
```

Comment:
why 2601?

Reply:
These were randomly picked previously and the number is passed in below. It made more sense before when the method calling it was above :) I'll pull it out into a variable and use it here as well as the call below.